src/components/tests/loop-form.test.tsx

```diff
@@ -15,59 +14,58 @@ const buildWrapper = (listOfPlaces?: string[]) => {
     console.log('test');
   };
 
-  const wrapper = mount(
-    <LoopForm
-      requestLoop={onClick}
-      fleets={['SuperFleet']}
-      listOfPlaces={!!listOfPlaces ? listOfPlaces : fakePlaces()['SuperFleet']}
-    />,
-  );
+  const wrapper = mount(<LoopForm requestLoop={onClick} fleets={[fleetName]} />);
   return wrapper;
 };
 
 describe('form Validation', () => {
+  afterEach(() => {
+    jest.clearAllMocks();
+  });
   test('Initial Values', () => {
-    const wrapper = buildWrapper();
+    const wrapper = buildWrapper('SuperFleet');
+    expect(wrapper.find('input[name="targetFleet"]').props().value != '');
     expect(wrapper.find("input[type='number']").props().value).toEqual('');
-    expect(wrapper.findWhere(x => x.name() === 'input' && x.props().value === 'supplies'))
-      .toBeTruthy;
-    expect(
-      wrapper.findWhere(
-        x => x.name() === 'input' && x.props().value === fakePlaces()['SuperFleet'][0],
-      ),
-    ).toBeTruthy;
-
-    expect(
-      wrapper.findWhere(
-        x => x.name() === 'input' && x.props().value === fakePlaces()['SuperFleet'][1],
-      ),
-    ).toBeTruthy;
+    expect(wrapper.findWhere(x => x.name() === 'input' && x.props().value != '')).toBeTruthy();
+    expect(wrapper.find('input[name="startLocation"]').props().value).not.toEqual(
+      wrapper.find('input[name="finishLocation"]').props().value,
+    );
     wrapper.unmount();
   });
 
+  test('Successful Request', () => {
+    const spy = jest.spyOn(console, 'log');
+    const wrapper = buildWrapper('SuperFleet');
+    wrapper.find("input[type='number']").simulate('change', { target: { value: 1 } });
+    expect(wrapper.find("input[type='number']").props().value).toEqual(1);
+    wrapper.find('form').simulate('submit');
+    expect(spy).toBeCalled();
+  });
+
   test('Number of loops cannot be empty', async () => {
-    const wrapper = buildWrapper();
+    const spy = jest.spyOn(console, 'log');
+    const wrapper = buildWrapper('SuperFleet');
     wrapper.find('form').simulate('submit');
-    expect(wrapper.find('#numLoopsError').html()).toContain('Loops can only be &gt; 0');
+    expect(spy).not.toBeCalled();
+    expect(wrapper.exists('.Mui-error')).toBeTruthy();
     wrapper.unmount();
   });
 
   test('Location cannot be empty', async () => {
-    const wrapper = buildWrapper(['placeA']);
+    const spy = jest.spyOn(console, 'log');
```

Comment:
This seems like a questionable spy. console.log can be called pretty much anywhere.